specs/windows/wmi_cli_event_consumers.table

```diff
@@ -0,0 +1,13 @@
+table_name("wmi_cli_event_consumers")
+description("WMI CommandLineEventConsumer, which can be used for persistance on Windows. https://www.blackhat.com/docs/us-15/materials/us-15-Graeber-Abusing-Windows-Management-Instrumentation-WMI-To-Build-A-Persistent%20Asynchronous-And-Fileless-Backdoor-wp.pdf")
+schema([
+    Column("name", TEXT, "Unique name of a consumer."),
+    Column("command_line_template", TEXT, "Standard string template that specifies the process to be started. This property can be NULL, and the ExecutablePath property is used as the command line."),
+    Column("executable_path", TEXT, "Module to execute. The string can specify the full path and file name of the module to execute, or it can specify a partial name. If a partial name is specified, the current drive and current directory are assumed."),
+    Column("__class", TEXT, "The name of the class."),
+    Column("__relpath", TEXT, "Relative path to the class or instance."),
```

Comment:
Would it be worth it for the column name to be something more readable? such as `relative_path`? Same for `class` above?